// @salutejs/client/esm/index.js
export { createAssistant } from './createAssistant.js';
export { createAssistantHostMock, createAssistantHostMockWithRecord } from './mock.js';
export { e as createAnswerFromMockByMessageGetter, f as createMockRecorder, d as createMockWSCreator, b as createOnlineRecordPlayer, a as createRecordPlayer } from './record-27e852e2.js';
export { createAssistantDev, createSmartappDebugger } from './createAssistantDev.js';
import { b as asrExports } from './assistant-d559dcae.js';
export { c as createAssistantClient, c as createAssistantSDK, a as createNavigatorAudioProvider } from './assistant-d559dcae.js';
import { p as protoExports } from './sdk-082d96e7.js';
export { M as MessageNames, V as VpsVersion } from './sdk-082d96e7.js';
import './common-ba25e019.js';



var GetHistoryResponse = protoExports.GetHistoryResponse;
var InitialSettings = protoExports.InitialSettings;
var PacketWrapperFromServer = asrExports.PacketWrapperFromServer;
export { GetHistoryResponse, InitialSettings, PacketWrapperFromServer };
